TypeCobol.Test/Parser/Programs/Cobol85/IncompleteIfCondition.rdzMix.txt

```diff
@@ -0,0 +1,19 @@
+﻿Line 1[8,23] <37, Warning, General> - Warning: "END PROGRAM" is missing.
```

Comment:
It's because there is `.` missing before `END-DECLARE`. It's another issue.

Reply:
In that case that, since the test is targeting a Cobol85 feature (as such it's in Programs/Cobol85), @delevoye  can you replace the existing test IncompleteIfCondition.rdzMix.txt with the following :
```cob
       IDENTIFICATION DIVISION.
       PROGRAM-ID. Pgm.
       ENVIRONMENT DIVISION.
       DATA DIVISION.
       WORKING-STORAGE SECTION.
       01 MyBool         pic X.
         88 O            value 'O'.
       PROCEDURE DIVISION.
           if O
           .
       END PROGRAM Pgm.
```